images/runtime/php-fpm/7.3/base.stretch.Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.2.* || $PHP_VERSION == 7.3.* ]]; then \
+    if [[ $PHP_VERSION == 7.3.* ]]; then \
```

Comment:
likewise, do we need this check if the image is for version 7.3 (from the file path)?